setup.py

```diff
@@ -1,14 +1,8 @@
 import glob
 import os
 import re
-import setuptools
 from pkg_resources import DistributionNotFound, get_distribution
-from setuptools import dist, find_packages, setup
-
-dist.Distribution().fetch_build_eggs(['Cython', 'numpy>=1.11.1'])
-
-import numpy  # NOQA: E402  # isort:skip
-from Cython.Build import cythonize  # NOQA: E402  # isort:skip
+from setuptools import find_packages, setup
 
 EXT_TYPE = ''
 try:
```

Comment:
Since no cython compilation is needed, the following logic can be simplified.

Reply:
Since we need to support non-torch compilation, I may think the logic should be kept as the previous one.

Reply:
We can just avoid the dependency of cython like the current implementation.